app/src/main/java/com/github/multimatum_team/multimatum/repository/FirebaseGroupRepository.kt

```diff
@@ -94,6 +94,27 @@ class FirebaseGroupRepository @Inject constructor(database: FirebaseFirestore) :
             .await()
     }
 
+    /**
+     * Rename a group.
+     * @param id the ID of the group to rename
+     * @param newName the new name of the group
+     */
+    override suspend fun rename(id: GroupID, newName: String) {
+        groupsRef
+            .document(id)
+            .update("name", newName)
+            .await()
+    }
+
+    /**
+     * Invite an user to join a group.
+     * @param id the ID of the grop to which we want to invite the user
+     * @param email the email of the user to invite
+     */
+    override suspend fun invite(id: GroupID, email: String) {
+        throw NotImplementedError("FirebaseGroupRepository.invite is not implemented")
```

Comment:
Same comment as above about the exception type

Reply:
> NotImplementedError suggests that the method is not yet implemented.

This is exactly what I meant in this case

Reply:
Oh OK, I thought you would not implement it at all